Makefile

```diff
@@ -28,13 +29,13 @@ lwt-clean:
 
 ## default tests
 %-configure:
-	$(MIRAGE) configure $*/config.ml --$(MODE) $(FLAGS)
+	$(MIRAGE) configure -f $*/config.ml -t $(MODE) $(FLAGS)
```

Comment:
I'm just stating the obvious, it's jut a not for myself (and other who are reading this, and maybe you if you forgot) to remember to check that before the release :-)